samples/bluetooth/path_loss_monitoring/README.rst

```diff
@@ -0,0 +1,135 @@
+.. _bluetooth_path_loss_monitoring:
+
+Bluetooth: Path Loss Monitoring
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Path Loss Monitoring sample demonstrates how to evaluate Bluetooth® LE signal quality using the Path Loss Monitoring feature. It consists of a Central and a Peripheral device. The Central continuously monitors the signal strength and classifies the connection quality using onboard LEDs.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+You must use two compatible development kits to run this sample: one programmed with the Central application, and one with the Peripheral application.
+
+Overview
+********
+
+This sample illustrates Bluetooth® LE Path Loss Monitoring by calculating the signal degradation between two connected devices. The Central application scans for a Peripheral named ``Path_Loss_Peripheral`` and connects to it. After a successful connection, the Central measures path loss using the formula:
+
+::
+
+   path loss = Tx Power - RSSI
+
+The application defines three signal quality zones based on the path loss value:
+
+- **Low path loss (strong signal):** LED0 is turned on.
+- **Medium path loss:** LED1 is turned on.
+- **High path loss (weak signal):** LED2 is turned on.
+
+The Peripheral simply advertises its presence and accepts connections.
+
+User interface
+**************
+
+Central:
+
+- **LED0:** Indicates **Low** path loss (good signal).
+- **LED1:** Indicates **Medium** path loss.
+- **LED2:** Indicates **High** path loss (weak signal).
+
+No buttons are used.
+
+Peripheral:
+
+- No direct user interface. The device advertises under the name ``Path_Loss_Peripheral``.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/path_loss_monitoring`
+.. include:: /includes/build_and_run.txt
+
+.. |sample_or_app| replace:: sample
+.. |ipc_radio_dir| replace:: :file:`sysbuild/ipc_radio`
+.. include:: /includes/ipc_radio_conf.txt
+
+Testing
+=======
+
+To test the sample:
+
+1. Program one DK with the **Peripheral Path Loss Monitoring** firmware.
+2. Program another DK with the **Central Path Loss Monitoring** firmware.
+3. Power both devices.
+4. The Central will start scanning and connect automatically to the Peripheral.
+5. Once connected, observe the LEDs on the Central to evaluate signal strength:
+   - Bring the devices closer or move them farther apart to change the signal quality.
+   - Optionally place an obstruction between the devices to see how this affects signal quality.
+6. Monitor the VCOM ports for logs showing connection status and real-time path loss.
```

Comment:
```suggestion
1. Program one DK with the **Peripheral Path Loss Monitoring** firmware.
#. Program another DK with the **Central Path Loss Monitoring** firmware.
#. Power on both devices.
#. The Central will start scanning and connect automatically to the Peripheral.
#. Once connected, observe the LEDs on the Central to evaluate signal strength:
   * Bring the devices closer or move them farther apart to change the signal quality.
   * Optionally place an obstruction between the devices to see how this affects signal quality.
6. Monitor the VCOM ports for logs showing connection status and real-time path loss.
```
By Peripheral and Central Path Loss Monitoring firmware, do you mean the samples? Which are?